docs/modules/ROOT/pages/editing.adoc

```diff
@@ -14,13 +14,20 @@ endif::[]
 
 toc::[]
 
-You can edit your script in an IDE/editor by using `jbang edit helloworld.java`. This will generate a project in a temporary location with symbolic links to your script and open a editor.
+You can edit your script in an IDE/editor by using `jbang edit helloworld.java`. This will generate a project in a temporary location with symbolic links to your script and open a editor. If your IDE supports JBang directly (see https://jbang.dev/ide) then you can open the folder instead and avoid the symbolic links and temporary location. That will generally be faster and more reliable.
```

Comment:
Is this explanation still relevant? Because by default the sandbox mode will never be used, right? Shouldn't it explain a bit that difference and suggest to use the sandbox option if you don't have an IDE with support for JBang?

Reply:
updated with bit different text

Reply:
Ok, and if I want to open that file directly (because it's a folder with a whole bunch of other scripts) with an IDE that supports JBang I'd have to use `jbang edit . helloworld.java`, right? It might be useful giving that example as well. Also, will `init` still say "use jbang edit newfile.java", etc? Because it would be nice if that actually told the user "use jbang edit . newfile.java" instead.